SPECS/podman/podman.spec

```diff
@@ -383,11 +340,11 @@ cp -pav test/system %{buildroot}/%{_datadir}/%{name}/test/
 %dir %{_libexecdir}/%{name}
 %{_libexecdir}/%{name}/gvproxy
 
-
 # rhcontainerbot account currently managed by lsm5
 %changelog
-* Tue Nov 01 2022 Olivia Crain <oliviacrain@microsoft.com> - 4.1.1-4
-- Bump release to rebuild with go 1.18.8
+* Tue Nov 01 2022 Ameya Usgaonkar <ausgaonkar@microsoft.com> - 4.1.1-4
```

Comment:
nit: changelog seems fishy

Reply:
While I had the above file deleted from EXTENDED and moved to SPECS in my branch, the above change happened.  I have brought in that change now.  Thanks for highlighting.